spec/javascripts/analytics/pii.spec.js

```diff
@@ -138,9 +138,44 @@ describe("GOVUK.PII", function() {
     });
   });
 
+  describe('when there is a govuk:static-analytics:strip-query-string-parameters meta tag present', function() {
+    afterEach(function() {
+      resetHead();
+    });
```

Comment:
Nitpick: some of your lines end with `;` and some don't. Looks like the file before changes was inconsistent already though. Would you mind stripping them out?

Reply:
Yup, no problem. Old habits die hard for me with JS semi-colons 😁 . I miss the linter, but hopefully I'll get it installed to this app in the next week or two.